build-system/tasks/build-bento.js

```diff
@@ -125,7 +124,6 @@ async function watchBentoComponent(
  * @return {!Promise<void|void[]>}
  */
 async function buildBentoComponent(name, version, hasCss, options = {}) {
-  options.npm = true;
```

Comment:
You're likely going to have to rebase this PR prior to merging, please ensure all the bundles correctly respect this at that point.